docs/userGuide/includingContents.md

```diff
@@ -25,11 +25,14 @@
 
     - `inline` (optional): make the included result an inline element. (wrapped in `<span>` tag)
 
+    - `optional` (optional): include the document only if it exists.
+
     Examples:
     ```html
     <include src="EstablishingRequirements.md#preview" inline/>
     <include src="../common/RequirementsVsSystemSpecification.md"/>
     <include src="../index.html" />
+    <include src="exercises.md" optional/>

```

Comment:
2. For the sake of consistency with the other examples, let's use a filename that suits the overall theme (e.g. `UserStories.md`).